### PR TITLE
Checksum everything

### PR DESCRIPTION
**Pull request by Darruma**

### CODE REVIEW

File: rewards/boost/calc_boost.py [hunk 1]
```diff
@@ -87,7 +88,7 @@ def badger_boost(current_block: int, chain: str):
 
     for addr, boost in badger_boost.items():
         boost_metadata = boost_info.get(addr, {})
-        boost_data[addr] = {
+        boost_data[Web3.toChecksumAddress(addr)] = {
```
Comment (mitche50) [inline rewards/boost/calc_boost.py:91]: Where is the source coming from on this?  To match further down, we should be checksumming the addr before saving in the `badger_boost` var

File: rewards/calc_rewards.py [hunk 2]
```diff
@@ -65,6 +72,7 @@ def fetch_all_schedules(chain: str, setts: List[str]):
     setts_with_schedules = []
     for sett in setts:
         schedules = logger.getAllUnlockSchedulesFor(sett).call()
+        sett = Web3.toChecksumAddress(sett)
```
Comment (mitche50) [inline rewards/calc_rewards.py:75]: should this go above the logger call?

Comment (Darruma) [reply rewards/calc_rewards.py:75]: Calling this function with lowercase vs checksum has no effect, web3 probably checksums this already. Its still good practice to make sure we checksum first anyway,

**Comment by mitche50** [comment]
Looks like we also have conflicts here - please merge and resolve them

File: rewards/boost/calc_boost.py (hunk 1 above)
Comment (Darruma) [reply rewards/boost/calc_boost.py]: This comes from the subgraph queries - these should be checksummed instead

File: rewards/boost/calc_boost.py [hunk 3]
```diff
@@ -43,6 +44,7 @@ def badger_boost(current_block: int, chain: str):
     )
 
     all_addresses = calc_union_addresses(native_setts, non_native_setts)
+    console.log(all_addresses)
```
Comment (mitche50) [inline rewards/boost/calc_boost.py:47]: Unnecessary log?

Comment (Darruma) [reply rewards/boost/calc_boost.py]: Yes

File: rewards/boost/calc_boost.py [hunk 4]
```diff
@@ -10,6 +10,7 @@
     calc_union_addresses,
     calc_boost_balances,
 )
+from web3 import Web3
```
Comment (mitche50) [inline rewards/boost/calc_boost.py:13]: unnecessary import here

**Review by mitche50 — APPROVED** [review]

**Pull request merged**